src/parse_api_responses_to_csv.py

```diff
@@ -12,8 +12,8 @@ def get_nested_output(key, resp, obj=None):
         return resp['outputs']['Scenario']['Site'][obj][val]
 
     else:
-        site = resp['outputs']['Scenario']['Site']
-
+        site = {k: v for k, v in resp['outputs']['Scenario']['Site'].items() if k.isupper()}
```

Comment:
@NLaws what is this trying to do? Should it be k[0].isupper()?

Reply:
It filters out all the Site attributes that are not capitalized, such as `latitude`.  I think that you're right. I'll test it locally.

Reply:
fixed with https://github.com/NREL/REopt-API-Analysis/pull/7/commits/fafa0a301cca8a8172dfc143bc0997f658d102b5